### ISSUE TITLE
Architect cannot find matrices that are stored in hd5 format. 

### ISSUE THREAD

**Issue by jesteria** [dssg/triage#280]
_From @avishekrk on August 31, 2017 16:14_

After switching matrices from csv to hd5. When running an experiment architect could not find any matrices. Looks like it is hardcoded to look for csv(s) and not matrices stored in hd5. 

_Copied from original issue: dssg/architect#38_

**Comment by jesteria** [comment]
_From @thcrock on August 31, 2017 16:42_

Yeah a couple things need to happen.  Here is an issue I created a while ago for tracking the implementation of hdf5 in architect (ie, building hdf matrices instead of the CSV implementation that's hardcoded).

https://github.com/dssg/architect/issues/18

In addition, we will need to implement a storage wrapper in catwalk similar to the MettaCSVMatrixStore that can be used in experiments. I created an issue for that here: https://github.com/dssg/catwalk/issues/17
